alerting/alert/alert.go

```diff
@@ -45,6 +46,9 @@ type Alert struct {
 	// ongoing/triggered incidents
 	ResolveKey string `yaml:"-"`
 
+	// ReminderInterval is the interval between reminders
+	ReminderInterval time.Duration `yaml:"reminder-interval,omitempty"`
```

Comment:
Call me super picky, but `reminder-interval` just doesn't sound right to me. I don't have a better name right now and I'm short on time, but I'd like to find a different name for that.

Reply:
@TwiN why not repeat-interval like in alertmanager?